torch/_dynamo/variables/user_defined.py

```diff
@@ -43,6 +47,14 @@ def var_getattr(self, tx, name: str) -> "VariableTracker":
         elif isinstance(obj, classmethod):
             return variables.UserMethodVariable(obj.__func__, self, **options)
 
+        if name in getattr(self.value, "__dict__", {}) or ConstantVariable.is_literal(
+            obj
+        ):
+            if source:
+                return VariableBuilder(tx, source)(obj).add_options(options)
+            elif ConstantVariable.is_literal(obj):
+                return ConstantVariable(obj, **options)
+
```

Comment:
This is not directly relevant to this PR, it's used to fix unit test failures exposed after we correct the above logic.